src/cmdc/datasets/uscensus/schema.sql

```diff
@@ -0,0 +1,71 @@
+DROP SCHEMA IF EXISTS uscensus CASCADE;
+
+CREATE SCHEMA uscensus;
+
+/* Create and define metadata tables */
+CREATE TABLE uscensus.acs_variables (
+    "id" SERIAL UNIQUE,
+    "year" INTEGER,
+    "product" CHAR(4),
+    "census_id" VARCHAR(20),
+    "label" VARCHAR(350),
+    PRIMARY KEY ("year", "product", "census_id")
```

Comment:
I think we may want to do this the other way around -- having the id be the primary key and creating a unique index/constraint on (year, product census_id)

I am not 100% sure, but there might be subtle differences in how tools (like postgraphile and postgrest) handle a foriegn key that is a primary key vs one that is only part of a unique constraint.

Reply:
Roger that. Fixed.